docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -95,7 +91,7 @@ listen ceph_rgw
   server controller-2.storage.redhat.local 172.17.3.156:8080 check fall 5 inter 2000 rise 2
 ----
 
-.. Confirm that the network is used as an HAProxy frontend:
+.. Confirm that the network is used as an HAProxy front end. The following example shows that `controller-0` exposes the services by using the external network, which is not present in the {Ceph} nodes. You must propagate the external network through {OpenStackPreviousInstaller}:
```

Comment:
Instead of "which is not present in the {Ceph} nodes"
could say "which is absent from the {Ceph} nodes"